client/src/components/App.jsx

```diff
@@ -45,6 +50,12 @@ export default class App extends React.Component {
       moreReviewBtn: reviews.length <= 2 ? false : true
       }))
     });
+
+    this.fetchMeta()
+    .then(data => {
+      var meta = data.data;
+      this.setState({meta})
```

Comment:
a shorter alternative is to destructure data and not define a new variable

({data}) =>{
  this.setState(data);
}

Reply:
I don't think It will work. I have to assign `data.data` value to state `meta` in the this.setState.

Reply:
ah gotcha. you could rename while destructuring. so you could do:
({data : meta}) =>{
this.setState({meta});
}